shiftcommerce/shift-next
# SHIFT SEARCH: Distinct is set from and ENV

We should pass in an ENV var for setting distinct on search. This allows an end user to implement their desired degree of distinction at runtime/deploy-time. 

closes #55 

## QA 

We should be able to set a config for this in a Next app
When this is not set we should expect the default value of true
